src/promptflow-devkit/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -155,6 +155,15 @@ def make_response_no_content():
     return make_response("", 204)
 
 
+def get_pfs_version():
+    version_promptflow = get_promptflow_sdk_version()
+    if version_promptflow:
+        return version_promptflow
+    else:
+        version_devkit = get_promptflow_devkit_version()
```

Comment:
why shall we do if else here?  I think devkit version should be good enough?

Reply:
according to the discussion before, we will show promptflow version if customer install from root which is also our recommendation installation method.